kratos/tests/test_container_expression.py

```diff
@@ -758,6 +758,20 @@ def _Read(self, container_expression, variable):
     def _Evaluate(self, container_expression, variable):
         Kratos.Expression.VariableExpressionIO.Write(container_expression, variable)
 
+    def testEntityDomainSizeExpressionIOCondition(self):
+        condition_exp = Kratos.Expression.ConditionExpression(self.model_part)
+        Kratos.Expression.EntityDomainSizeExpressionIO.Read(condition_exp)
+        numpy_condition_exp = condition_exp.Evaluate()
+        for i, condition in enumerate(self.model_part.Conditions):
+            self.assertAlmostEqual(numpy_condition_exp[i], condition.GetGeometry().DomainSize(), 12)
+
+    def testEntityDomainSizeExpressionIOElement(self):
+        element_exp = Kratos.Expression.ElementExpression(self.model_part)
+        Kratos.Expression.EntityDomainSizeExpressionIO.Read(element_exp)
+        numpy_element_exp = element_exp.Evaluate()
+        for i, condition in enumerate(self.model_part.Elements):
+            self.assertAlmostEqual(numpy_element_exp[i], condition.GetGeometry().DomainSize(), 12)
```

Comment:
please make the elements'/conditions' doman sizes different, otherwise the tests will pass even if the IO gets the wrong entity. Also, add tests for empty containers.

Reply:
and in line 772, you're looping over elements, not conditions

Reply:
I am iterating in different containers here. I have a naming problem which is fixed.
Added empty mp tests.